inc/class-cachify.php

```diff
@@ -422,15 +422,16 @@ private static function _get_options() {
 		return wp_parse_args(
 			get_option( 'cachify' ),
 			array(
-				'only_guests'      => 1,
-				'compress_html'    => self::MINIFY_DISABLED,
-				'cache_expires'    => 12,
-				'without_ids'      => '',
-				'without_agents'   => '',
-				'use_apc'          => self::METHOD_DB,
-				'reset_on_post'    => 1,
-				'reset_on_comment' => 0,
-				'sig_detail'       => 0,
+				'only_guests'       => 1,
+				'compress_html'     => self::MINIFY_DISABLED,
+				'cache_expires'     => 12,
+				'without_ids'       => '',
+				'without_agents'    => '',
+				'use_apc'           => self::METHOD_DB,
+				'reset_on_post'     => 1,
+				'reset_on_comment'  => 0,
+				'sig_detail'        => 0,
+				'change_robots_txt' => 0,
```

Comment:
I suggest enabling this feature by default, so we do not silently disable it when upgrading existing installations.

```suggestion
				'change_robots_txt' => 1,
```

Reply:
I disabled it by default because the last state caused a broken setup. Now with #283 merged it should be fine to change it back.